app/Core/Repository/ServiceRepository.cs

```diff
@@ -86,6 +86,76 @@ public Services GetService(Services service)
             return null;
         }
 
+        public bool IsDuplicateServiceCode(string serviceCode)
+        {
+            string query = "SELECT COUNT(*) FROM services WHERE serviceCode = @ServiceCode AND isDeleted = 0";
+
+
+            Dictionary<string, object> parameters = new Dictionary<string, object>
+    {
+        { "@ServiceCode", serviceCode }
+    };
+
+
+            UpgradeFile upgradeFile = new UpgradeFile();
+
+
+            int count = upgradeFile.ExecuteScalar(query, parameters);
+
+            return count > 0;
+        }
+
+
+        public bool UpdateServicePrice(string serviceCode, decimal newPrice)
```

Comment:
Does this method still needed, I don't see any code or block that reference in this method. Remove it if it has no use.